events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -779,176 +784,204 @@
 
     function onWebEventReceived(event) {
         if (typeof event === 'string') {
-            event = JSON.parse(event);
+            try {
+                event = JSON.parse(event);
+            } catch (e) {
+                console.log("error parsing web event", e);
+                return;
+            }
             if (event.app === 'trivia') {
-                switch (event.type) {
-                    case 'catalog':
-                        switch (event.value) {
-                            case "Default Catalog":
-                                useGoogle = false;
-                                break;
-                            case "Custom Catalog":
-                                useGoogle = true;
-                                triviaURL = SECRETS.trivia_URL;
-                                answer = Window.prompt("What is the tab name of the sheet to use?", "");
-                                if (answer === null) {
-                                    tablet.emitScriptEvent("nothingEntered");
-                                    useGoogle = false;
-                                } else {
-                                    gSheetTabName = answer;
-                                }
-                                break;
-                            case "Misc. Catalog":                                
-                                var answer = Window.prompt("URL of your GSheets script?", "");
-                                if (answer === null) {
-                                    tablet.emitScriptEvent("nothingEntered");
+                var newTime = new Date();
+                if (newTime - lastTime > HFC_INCREMENT) {
```

Comment:
this is not a sustainable solution. if you are very confident that this is guaranteed to not cause issues, keep this (although please change the name of `HFC_INCREMENT`; i don't think that's a descriptive var name for a timeout), and then put a "known issue" in the README that talks about how there is an artificial delay in web event message processing.

if you are not 100% confident this won't cause issues, please figure out why your web JS is sending multiple events and fix that bug. don't spend more than 2 hours on it.